src/protocol/DocumentServer.js

```diff
@@ -222,6 +223,33 @@ define('polymer-designer/protocol/DocumentServer', [
       });
     }
 
+    backspaceText(request) {
+      let text = request.message.text;
+      let node = document.createTextNode(text);
```

Comment:
That is very strange, you are not using this `node` anywhere, you are writing a new value to `node` on line 238. What happens if you rename `node` to `what`? Is the simple act of creating a Text node causing something else to happen?